data/src/main/java/com/linkedin/data/schema/resolver/ClasspathResourceDataSchemaResolver.java

```diff
@@ -80,6 +79,26 @@ public ClasspathResourceDataSchemaResolver(ClassLoader classLoader)
     _classLoader = classLoader;
   }
 
+  /**
+   * Construct a new instance that uses the specified {@link ClassLoader}.
+   *
+   * @param classLoader provides the {@link ClassLoader}.
+   * @param schemaDirectoryName The file directory name for different types of schemas.
+   *                            Default is {@link SchemaDirectoryName#PEGASUS}
+   */
+  public ClasspathResourceDataSchemaResolver(ClassLoader classLoader, SchemaDirectoryName schemaDirectoryName)
```

Comment:
So for our case, the resolver we use should be this one and only one right?
And this is the resolver that be passed to the parser.

Reply:
Both classpath or extension resolver can be used depending on build time or runtime